### ISSUE TITLE
Dont work

### ISSUE THREAD

**Issue by ugc-gaming** [skial-com/port64#2]
Hey,

Does this even work, can you fix this fast?, Thank you.

**Comment by ugc-gaming** [comment]
Dont work, makes my servers crash, dont use this shit

**Comment by ghost** [comment]
I don't think this is an appropriate way of addressing your concerns with this extension.

Have you looked into why servers crash with this extension? My assumption is that the `Port64_StoreToAddress` native does not set the memory access of the memory region it is modifying, therefore a memory exception is thrown by the MMU.
